src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1180,6 +1181,19 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
             $translator->replaceArgument(4, $options);
         }
+
+        if ($config['pseudo_localization']['enabled']) {
+            $options = $config['pseudo_localization'];
+            unset($options['enabled']);
+
+            $container
+                ->register(PseudoLocalizationTranslator::class)
```

Comment:
don't use the class name as service id. We don't want the PseudoLocalizationTranslator type to be available in autowiring.
I suggest using `translator.pseudo` as service id.